Source/CarthageKit/BuildOptions.swift

```diff
@@ -16,20 +16,24 @@ public struct BuildOptions {
 	public var cacheBuilds: Bool
 	/// Whether to use downloaded binaries if possible.
 	public var useBinaries: Bool
+	/// Whether to create an XCFramework instead of lipoing built products.
+	public var createXCFramework: Bool
 
 	public init(
 		configuration: String,
 		platforms: Set<SDK>? = nil,
 		toolchain: String? = nil,
 		derivedDataPath: String? = nil,
 		cacheBuilds: Bool = true,
-		useBinaries: Bool = true
+		useBinaries: Bool = true,
+		createXCFramework: Bool = false
```

Comment:
see here

Reply:
I think the only place this default value is actually used is in unit tests (see my comment above)